src/UndisclosedPassword.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+
+namespace Zend\Validator;
+
+use Psr\Http\Client\ClientExceptionInterface;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
+use Psr\Http\Message\ResponseFactoryInterface;
+
+final class UndisclosedPassword extends AbstractValidator
+{
+    const HIBP_API_URI = 'https://api.pwnedpasswords.com';
```

Comment:
@weierophinney all of these constants should be `private` in order to prevent needless BC breaks later on: can we please bump to `^7.1` before expanding the API surface beyond what should be `public`?